src/nix/meson.build

```diff
@@ -68,6 +68,21 @@ nix_src = files(
     'upgrade-nix.cc',
     'verify.cc',
     'why-depends.cc')
+
+src_markdown_files = files(
+    'flake-lock.md', 'store-repair.md', 'nix.md', 'why-depends.md', 'registry-list.md', 'store-dump-path.md',
+    'nar.md', 'edit.md', 'nar-ls.md', 'store-delete.md', 'nar-cat.md', 'upgrade-nix.md', 'show-derivation.md',
+    'copy.md', 'bundle.md', 'registry.md', 'search.md', 'optimise-store.md', 'key-convert-secret-to-public.md',
+    'profile-install.md', 'run.md', 'nar-dump-path.md', 'flake-archive.md', 'profile.md', 'flake-check.md',
+    'daemon.md', 'repl.md', 'store-prefetch-file.md', 'profile-upgrade.md', 'add-path.md', 'flake-new.md',
+    'profile-history.md', 'profile-diff-closures.md', 'help.md', 'diff-closures.md', 'key-generate-secret.md',
+    'store-gc.md', 'registry-add.md', 'profile-remove.md', 'shell.md', 'path-info.md', 'ping-store.md',
+    'flake.md', 'flake-update.md', 'flake-clone.md', 'eval.md', 'flake-metadata.md', 'profile-list.md',
+    'flake-init.md', 'make-content-addressable.md', 'registry-pin.md', 'verify.md', 'develop.md',
+    'flake-prefetch.md', 'store-ls.md', 'add-file.md', 'flake-show.md', 'build.md', 'registry-remove.md',
+    'print-dev-env.md', 'store-cat.md', 'log.md', join_paths('realisation', 'info.md'),
+)
+
```

Comment:
could you please newline these file names to improve readability?

Reply:
Although I generally agree, I don't for that file:
There is a substantial amount of source files that are declared, forcing to scroll quite a bit already to go to the interesting part (IMO, `executable` declarations are more important than lines and lines of files).